tests/unit/test_readme_missing_namespace.py

```diff
@@ -11,4 +12,5 @@ def test():
     for env in ["development", "test"]:
         pipeline.add_children(job_for(env))
 
-    conftest.check(pipeline.render())
+    with pytest.raises(ValueError, match="NAMING CONFLICT: Two jobs have the same name"):
```

Comment:
If we would have a custom exception `JobNameError` you could to 
```
with pytest.raises(JobNameError)
```

Reply:
solved bb569c5